<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CharacterCreator</class>
 <widget class="QMainWindow" name="CharacterCreator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>576</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>810</width>
    <height>576</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>810</width>
    <height>576</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>810</width>
    <height>576</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ProcjetAdventure</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>531</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinAndMaxSize</enum>
     </property>
     <item row="0" column="0">
      <layout class="QFormLayout" name="formLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item row="1" column="0">
        <widget class="QLabel" name="strLabel">
         <property name="text">
          <string>Strength</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="strScore">
         <property name="text">
          <string>Score</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="dexLabel">
         <property name="text">
          <string>Dexterity</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="dexScore">
         <property name="text">
          <string>Score</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="conLabel">
         <property name="text">
          <string>Constitution</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="conScore">
         <property name="text">
          <string>Score</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="wisLabel">
         <property name="text">
          <string>Wisdom</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="wisScore">
         <property name="text">
          <string>Score</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="intLabel">
         <property name="text">
          <string>Intelligence</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="intScore">
         <property name="text">
          <string>Score</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="chaLabel">
         <property name="text">
          <string>Charisma</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLabel" name="chaScore">
         <property name="text">
          <string>Score</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QPushButton" name="generateButton">
         <property name="text">
          <string>Generate stats</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="chaName">
         <property name="text">
          <string>Character Name:</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QPushButton" name="continueButton">
         <property name="text">
          <string>Continue</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QTextEdit" name="chaNameBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>28</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Stats</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>810</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
